DemoServer/AdditionalLanguages/python/basics/deleteDocument/metadata.json

```diff
@@ -0,0 +1,47 @@
+{
+    "Slug": "delete-document",
+    "SourceFileName": "deleteDocument.py",
+    "Title": "Delete Document",
+    "DescriptionLines": [
+        "**Delete a document** from the database."
+    ],
+
+    "Assets": [],
+
+    "Walkthroughs": [
+        {
+            "Title": "Mark the entity to be deleted",
+            "Slug" : "step-1",
+            "DescriptionLines": [
+                "* Mark the entity to be deleted with the Session `delete()` method.<br>",
+                "  The document is actually deleted from the database only when save_changes() is called later on.",
+                "",
+                "_Note_:",
+                "",
+                "* When using delete() as above, with the specific ID param overload, the document is deleted with no concurrency checks.",
```

Comment:
we should avoid the 'overload' word, to be pure

in Python, there are no overloads - we either check the argument type passed to the method or define two different methods with different names like `delete_by_id` or `delete_by_entity `

in our case, `delete()` takes `key_or_entity` argument

```py
def delete(self, key_or_entity: Union[str, object], expected_change_vector: Optional[str] = None) -> None:
        if isinstance(key_or_entity, str):
```

Reply:
done